Client/Frontend/Menu/AppMenuConfiguration.swift

```diff
@@ -73,7 +73,11 @@ struct AppMenuConfiguration: MenuConfiguration {
     private func numberOfMenuItemsPerRowForAppState(appState: AppState) -> Int {
         switch appState {
         case .TabTray:
-            return 4
+            if #available(iOS 9, *) {
```

Comment:
this should not be neccessary and the fact that it is shows up a bug in the menu code - it should display up to `numberOfMenuItemsPerRowForAppState` correctly, not leave space for the 4th when only 3 items are present. I've raised a bug and submitted a PR for this here https://bugzilla.mozilla.org/show_bug.cgi?id=1267581